src/app/shared/styles/elements/containers.scss

```diff
@@ -1,6 +1,5 @@
 .argo-container {
     position: relative;
-    width: $basePageWidth + 160;
```

Comment:
The .argo-container  and fixed-width are meant to limit width. Instead of removing width here we should just stop using .argo-container in places where it is not appropriate.

Reply:
ok, let me try that